CONTRIBUTING.md

```diff
@@ -1,5 +1,8 @@
 # Contributing
 
+This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to uphold this code.
```

Comment:
I think we can choose a better word than "uphold". First, it may not translate well for a non-native speaker (even being a native speaker I have to think how to apply it here). Second, it's more saying that you support or advocate against a challenge rather than something you apply. We want the focus to be on the application of the code.

The most fitting replacement that comes to my mind is "abide by", though I recognize that's can come across as rather strong. Other options include "respect", "honor" or "practice". In casual speech, I've always heard my elders use the word "honor" in this context.